server/controllers/network-sheet-parser.js

```diff
@@ -306,9 +310,15 @@ exports.workbookType = function (workbookFile) {
             const cellA1 = sheet.data[0][0];
 
             if (cellA1 === CELL_A1_GRN) {
+<<<<<<< HEAD
+                workbookType = NETWORK_GRN_MODE;
+            } else if (cellA1 === CELL_A1_PPI) {
+                workbookType = NETWORK_PPI_MODE;
+=======
                 workbookType = "grn";
             } else if (cellA1 === CELL_A1_PPI) {
                 workbookType = "protein-protein-physical-interaction";
+>>>>>>> beta
```

Comment:
Hmmm…either this is a missed commit or a missed test-run before pushing 😅

Reply:
I don't really know what issue. It passed all the tests and lint, and pass merge conflict too. Fixed